src/wrappers/node.js

```diff
@@ -1 +1,2 @@
+global.FormData = require('form-data');
```

Comment:
on the client it's `window`, in node it's the global state (`_self` I think)... it's provided by webpack when it's building the script